html5/js/DecodeWorker.js

```diff
@@ -211,30 +199,35 @@ function decode_draw_packet(packet, start) {
       // broadway decoding is actually synchronous
       // and onPictureDecoded is called from decode(data) above.
       if (count == 0) {
-        decode_error("no " + coding + " picture decoded");
+        decode_error(`no ${coding} picture decoded`);
       }
     } else {
       //pass-through:
       send_back([]);
     }
-  } catch (e) {
+  } catch (error) {
     decode_error(
-      "error processing " + coding + " packet " + packet_sequence + ": " + e
+      `error processing ${coding} packet ${packet_sequence}: ${error}`
     );
   }
 }
 
-function check_image_decode(format, image_bytes, success_cb, fail_cb) {
+function check_image_decode(
+  format,
+  image_bytes,
+  success_callback,
+  fail_callback
+) {
```

Comment:
The linter is saying "`cb` should be un-abbreviated to `callback`. Oh, a >80-character line? Time for newlines."

To me, this looks right. Plus, it's a >40-line function, so this doesn't really make it top-heavy. However, I'm open to either simplifying the arguments or extending the line cutoff from 80 characters to something like 100 characters (but only extending if this newline-ing is happening a lot and annoyingly).

@andersevenrud: What do you think of this auto-change?

Reply:
I don't really have any strong opinions on aestetics, so either way is fine by me :sweat_smile: I do have some minor reading issues though, so breaking down longer lines like this actually is prefereable in some cases.

Personally, I just use the recommended rulesets (with some very minor exceptions) and let the formatter do whatever it needs to, because in the end it doesn't really matter.

Reply:
I'd prefer to leave this as-is, since I think it's an acceptable result of the intersection of multiple good rules.